processor/probabilisticsamplerprocessor/README.md

```diff
@@ -135,7 +138,67 @@ In order for hashing to be consistent, all collectors for a given tier
 at different collector tiers to support additional sampling
 requirements.
 
-This mode uses 14 bits of sampling precision.
+This mode uses 14 bits of information in its sampling decision; the
+default `sampling_precision`, which is 4 hexadecimal digits, exactly
+encodes this information.
+
+### Proportional
+
+OpenTelemetry specifies a consistent sampling mechanism using 56 bits
+of randomness, which may be obtained from the Trace ID according to
+the W3C Trace Context Level 2 specification.  Randomness can also be
+explicly encoding in the OpenTelemetry `tracestate` field, where it is
+known as the R-value.
+
+This mode is named because it reduces the number of items transmitted
+proportionally, according to the sampling probability.  In this mode,
+items are selected for sampling without considering how much they were
+already sampled by preceding samplers.
+
+This mode is selected when `mode` field is unset and the `hash_seed`
+field is not set and for logs records when the `attribute_source` is
+`traceID`.
+
+This mode uses 56 bits of information in its calculations.  The
+default `sampling_precision` (4) will cause thresholds to be rounded
+in some cases when they contain more than 16 significant bits.
+
+### Equalizing
+
+This mode uses the same randomness mechanism as the propotional
+sampling mode, in this case considering how much each item was already
+sampled by preceding samplers.  This mode can be used to lower
+sampling probability to a minimum value across a whole pipeline, which
+has the effect of increasing trace completeness.
```

Comment:
I've added a use-cases sub-section to each sampler mode, explaining why they might be useful. However, I removed the sentence about completeness.

However, I found it unnecessarily torturous to construct an example where completeness is helped, because it requires the user to first make a departure from `ParentBased` sampler configuration in the SDK. As long as SDKs use `ParentBased`, traces should be complete regardless of what this component does.

It's when the use `TraceIDRatioBased` at intermediate nodes in a trace, that the equalizing mode could possibly be useful, but I don't want to go into this detail. a98db61504